src/demo/app/app.module.ts

```diff
@@ -6,11 +6,20 @@ import { HttpClientModule } from '@angular/common/http';
 import { AppComponent } from './app.component';
 import { DemoComponent } from './demo.component';
 import { AppRoutingModule } from './app-routing.module';
+import { NgxKeyboardShortcutsModule } from '../../../dist';
```

Comment:
changes to the demo files here in this branch seems out of place.  Can we do that as a different branch?